lib/tasks/test-server.js

```diff
@@ -8,9 +8,15 @@ const SilentError = require('silent-error');
 class TestServerTask extends TestTask {
   invokeTestem(options) {
     let task = this;
-
     return new Promise((resolve, reject) => {
-      task.testem.startDev(task.testemOptions(options), (exitCode, error) => {
+      if (task.testem.setDefaultOptions) {
+        task.testem.setDefaultOptions(task.testemOptions(options));
+      } else {
+        // To make it backward compatibile with older versions of testem (2.0.0 and lower)
```

Comment:
I don't think we need this branch, as ember-cli will need to bump its minimum testem version to one that supports this feature